packages/remediations/src/NewRemediationWizard.js

```diff
@@ -1,28 +1,37 @@
-import React, { Component, Fragment, createContext } from 'react';
+import React, { Component, Fragment } from 'react';
 import keyBy from 'lodash/keyBy';
 import transform from 'lodash/transform';
 import FormRenderer from '@data-driven-forms/react-form-renderer/dist/esm/form-renderer';
 import Pf4FormTemplate from '@data-driven-forms/pf4-component-mapper/dist/esm/form-template';
 import schemaBuilder from './schema';
+import { remediationUrl } from './utils';
+import * as api from './api';
 import Deferred from '@redhat-cloud-services/frontend-components-utilities/files/Deffered';
 import Wizard from '@data-driven-forms/pf4-component-mapper/dist/esm/wizard';
 import componentTypes from '@data-driven-forms/react-form-renderer/dist/esm/component-types';
 import SelectPlaybook from './newSteps/selectPlaybook';
 import ReviewActions from './newSteps/reviewActions';
 import IssueResolution from './newSteps/issueResolution';
-import * as api from './api';
-
-const RemediationWizardContext = createContext({
-    success: false,
-    submitting: false,
-    error: undefined
-});
+import FetchError from './newSteps/fetchError';
+import Review from './newSteps/review';
+
+function createNotification (id, name, isNewSwitch) {
+    const verb = isNewSwitch ? 'created' : 'updated';
+
+    return {
+        variant: 'success',
+        title: `Playbook ${verb}`,
+        description: <span>You have successfully { isNewSwitch ? 'created' : 'updated' } <a href={ remediationUrl(id) } >{ name }</a>.</span>,
+        dismissable: true
+    };
+}
 
 class RemediationWizard extends Component {
```

Comment:
One thing that we should definitely do before actually deploying this is also refactor the class component to the functional component. Does not have to be in this PR but in a follow-up.

Reply:
Sure, I'll finish this one and take a look :-)